src/Products/PloneMeeting/tests/helpers.py

```
@@ -644,3 +644,14 @@ def _setupFollowUp(self, cfg):
         new_config['edit_access_on_cache'] = "0"
         config.append(new_config)
         cfg.setLabelsConfig(config)
+
+    def _setupItemFieldsConfig(self, field_name, cfg=None, view=None, edit=None):
+        """Configure MeetingConfig.itemFieldsConfig."""
+        cfg = cfg or self.meetingConfig
+        config = cfg.getItemFieldsConfig()
+        field_config = [row for row in config if row['name'] == field_name][0]
```

Ruff (0.15.10)

[warning] Prefer `next(row for row in config if row['name'] == field_name)` over single element slice

Replace with `next(row for row in config if row['name'] == field_name)`

(RUF015)